lib/ui/discover/discover_page.dart

```diff
@@ -1,13 +1,91 @@
 import 'package:farmsmart_flutter/model/loading_status.dart';
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
 import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
 import 'package:farmsmart_flutter/data/model/article_entity.dart';
 import 'package:farmsmart_flutter/ui/discover/discover_child_item.dart';
 import 'package:farmsmart_flutter/ui/discover/discover_viewmodel.dart';
 import 'package:farmsmart_flutter/redux/home/discover/discover_actions.dart';
 
+abstract class HomeDiscoverPageStyle {
+  final Color separatorColor;
+
+  final TextStyle titlePageStyle;
+  final TextStyle titleArticleStyle;
+  final TextStyle summaryArticleStyle;
+
+  final EdgeInsets titlePadding;
+  final EdgeInsets headerArticlePadding;
+  final EdgeInsets itemListPadding;
```

Comment:
lets extract out the list items styles to the list item widget and just have an entry here for the list item style

Reply:
Now I split it in two styles, one for ArticleList and other one for ArticleListItem